hack/log/redact.sh

```diff
@@ -28,11 +28,7 @@ echo "================ REDACTING LOGS ================"
 
 log_files=( $(find "${ARTIFACTS:-${PWD}/_artifacts}" -type f) )
 redact_vars=(
-    "${AZURE_CLIENT_ID}"
```

Comment:
we should still redact these as best effort if they do get set for any reason

maybe redirect stdout to /dev/null so it doesn't log "not found" everytime